autoarray/plot/wrap/two_d/grid_errorbar.py

```diff
@@ -25,6 +25,12 @@ class GridErrorbar(AbstractMatWrap2D):
         list of colors can be specified which the plot cycles through.
     """
 
+    def config_dict_remove_marker(self, config_dict):
+        if config_dict.get("fmt") and config_dict.get("marker"):
+            config_dict.pop("marker")
+
+        return config_dict

```

Comment:
Conditionally mutating a dictionary is a bit nasty. Consider creating a new dictionary:

```python
return {key: value for key, value in config_dict.items() if key != "marker"}